pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/MailTemplateDao.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.ddp.db.dao;
+
+import org.broadinstitute.ddp.db.dto.MailTemplateDto;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindBean;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.util.Optional;
+
+public interface MailTemplateDao extends SqlObject {
+    @SqlQuery("SELECT * FROM `mail_template` WHERE mail_template_id = :id")
+    @RegisterConstructorMapper(MailTemplateDto.class)
+    Optional<MailTemplateDto> findById(@Bind("id") final Long id);
```

Comment:
Should this be `long` instead of `Long`? I don't see any cases where `findById(null)` would return a useful result.